posix.mak

```diff
@@ -124,7 +124,7 @@ STYLES=$(addsuffix .css, $(addprefix css/, \
 PRETTIFY=prettify/prettify.css prettify/prettify.js
 
 PREMADE=appendices.html articles.html fetch-issue-cnt.php	\
-howtos.html language-reference.html robots.txt process.php
+howtos.html language-reference.html robots.txt .htaccess process.php
```

Comment:
win32.mak can't build many parts of the website anyway, not nearly enough as needed to the point where you can start it in Apache and get a dlang.org instance.